src/data-fetchers/bed/shared-types.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * All data stored in each BED file eventually gets put into this
+ */
+export interface BedTile {
+    chrom: string;
+    chromStart: number;
+    chromEnd: number;
+    name?: string;
+    score?: number;
+    strand?: string;
+    thickStart?: number;
+    thickEnd?: number;
+    itemRgb?: string;
+    blockCount?: number;
+    blockSizes?: number[];
+    blockStarts?: number[];
+    [customField: string]: string | number | number[] | undefined;
+}
+
+export interface EmptyTile {
+    tilePositionId: string;
+}
```

Comment:
Is this used in somewhere?

Reply:
It was being used in the bed fetcher. I think it'll be better if I move it this interface to the worker so I'll do that.